src/app/project-notification/documents/project-notification-documents.component.ts

```diff
@@ -100,13 +105,15 @@ export class ProjectNotificationDocumentsComponent implements OnInit, OnDestroy
       .subscribe(({ documents }: any) => {
         if (documents) {
           if (documents[0].data && documents[0].data.meta.length > 0) {
-            this.tableParams.totalListItemsCategorized = documents[0].data.meta[0].searchResultsTotal;
             this.docs = documents[0].data.searchResults;
           } else {
-            this.tableParams.totalListItemsCategorized = 0;
             this.docs = [];
           }
 
+          this.tableParams.sortBy = '-datePosted';
+          this.tableParams.pageSize = 10;
+          this.tableParams.currentPage = 1;
+          this.tableParams.totalListItems = documents[0].data.meta[0].searchResultsTotal;
```

Comment:
PN Documents page doesn't load if the project doesn't have any documents

Reply:
Fixed in https://github.com/bcgov/eagle-admin/pull/718/commits/b68fd7c87f5c415b5f84f0b3202eccdb9e6d0768